corehq/apps/reports/tasks.py

```diff
@@ -183,8 +184,7 @@ def monthly_reports():
 
 @periodic_task(run_every=crontab(hour=[22], minute="0", day_of_week="*"), queue=getattr(settings, 'CELERY_PERIODIC_QUEUE','celery'))
 def saved_exports():
-    for group_config in HQGroupExportConfiguration.view("groupexport/by_domain", reduce=False,
-                                                        include_docs=True).all():
+    for group_config in get_all_hq_group_export_configs:
```

Comment:
Yup; updated

On Tue, Dec 1, 2015 at 6:11 PM, Daniel Roberts <notifications@github.com>
wrote:

> In corehq/apps/reports/tasks.py
> <https://github.com/dimagi/commcare-hq/pull/9293#discussion_r46354439>:
>
> > @@ -183,8 +184,7 @@ def monthly_reports():
> >
> >  @periodic_task(run_every=crontab(hour=[22], minute="0", day_of_week="*"), queue=getattr(settings, 'CELERY_PERIODIC_QUEUE','celery'))
> >  def saved_exports():
> > -    for group_config in HQGroupExportConfiguration.view("groupexport/by_domain", reduce=False,
> > -                                                        include_docs=True).all():
> > +    for group_config in get_all_hq_group_export_configs:
>
> Also this will definitely break, right? You need to call
> get_all_hq_group_export_configs
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/dimagi/commcare-hq/pull/9293/files#r46354439>.
>